### DISCUSSION TITLE
The future of next.js?

### DISCUSSION THREAD

**Discussion by reboxer** [vercel/next.js#12992] [Help]
We were about to start a new project using next, but now with the recent acquisition of zeit, we are afraid of the license suddenly changing and being stuck with an old fork of next that we would have to maintain ourselves.
Will the license ever change and become paid? will next remain free for anyone to use for ever?

**Comment by Timer** [comment]
ZEIT was not acquired, only renamed. You can read more about this here:
https://vercel.com/blog/zeit-is-now-vercel

**Reply by reboxer** [reply]
It seems that i've misunderstood the article
So we should not worry about this, as next will stay the same as always and free to use for everyone, right?

**Reply by adamsoffer** [reply]
ZEIT rebranded and raised money. If anything your confidence in the framework and its future prospects should be reinforced.
